dev-tools/mage/integtest.go

```diff
@@ -326,11 +326,15 @@ func dockerComposeBuildImages() error {
 		return err
 	}
 
-	args := []string{"-p", dockerComposeProjectName(), "build", "--pull", "--force-rm"}
+	args := []string{"-p", dockerComposeProjectName(), "build", "--force-rm"}
 	if _, noCache := os.LookupEnv("DOCKER_NOCACHE"); noCache {
 		args = append(args, "--no-cache")
 	}
 
+	if _, forcePull := os.LookupEnv("DOCKER_PULL"); forcePull {
+		args = append(args, "--pull")
+	}
```

Comment:
These changes shouldn't be needed then if we are not using the pre-cached oracle image, right?

Reply:
https://github.com/elastic/beats/pull/12890#issuecomment-598094258

I'd keep then the default behavior it is the same as before, but in the CI we can disable the pull and use the packer cache, otherwise, it is not possible to use the cache.

Reply:
Ok. In any case take into account that cache will still be used, layers that are already present won't be dowloaded again.